recipes/pyuoi/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "pyuoi" %}
+{% set version = "0.0.61" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4085faf3c6dd2add38fb199350c50fcccbe6eafe78c07ec25844ca4d773d7095
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - cython
+  run:
+    - python
+    - h5py
+    - scikit-learn
+
+test:
+  imports:
+    - pyuoi
+
+about:
+  home: https://github.com/BouchardLab/pyuoi
+  license: BSD
```

Which BSD licence is it? The most common is this but please check 
```suggestion
  license: BSD-3-Clause
```